textdb/textdb-exp/src/main/java/edu/uci/ics/textdb/exp/nltksenti/NltkSentiOperator.java

```diff
@@ -0,0 +1,183 @@
+                inputAttributeType.equals(AttributeType.TEXT);
+        if (! isValidType) {
+            throw new RuntimeException(String.format(
+                    "input attribute %s must have type String or Text, its actual type is %s",
+                    predicate.getInputAttributeName(),
+                    inputAttributeType));
+        }
+        
+        // generate output schema by transforming the input schema
+        outputSchema = transformSchema(inputOperator.getOutputSchema());
+        
+        cursor = OPENED;
+	}
+	
+	public static void main(String[] args2){
+	    String className;
+	    String inputText = "I am glad you arrived!! :)";
+        
+        try{
+            String path = Utils.getResourcePath("Python", TextdbProject.TEXTDB_EXP);
+            path += PYTHONSCRIPT;
+            String trainner = Utils.getResourcePath("Senti.pickle", TextdbProject.TEXTDB_EXP);
+            List<String> args = new ArrayList<String>(Arrays.asList(PYTHON, path, trainner));
+            args.addAll(new ArrayList<String>(Arrays.asList(inputText)));
+            System.out.println(args);
+
+            ProcessBuilder pb = new ProcessBuilder(args);
+            
+            Process p = pb.start();
+            int exitcode = p.waitFor();
+            
+            BufferedReader input = new BufferedReader(new InputStreamReader(p.getInputStream()));
+            if ((className = input.readLine()) != null) {
+                input.close();
+                System.out.println(className);
+            }
+        } catch(Exception e){
+            e.printStackTrace();
+            System.out.print(e.getMessage());
+        }
+	}
+	@Override
+	public Tuple getNextTuple() throws TextDBException {
+	    if (cursor == CLOSED) {
+            return null;
+        }
+        Tuple inputTuple = inputOperator.getNextTuple();
+        if (inputTuple == null) {
+            return null;
+        }
+        
+        List<IField> outputFields = new ArrayList<>();
+        outputFields.addAll(inputTuple.getFields());
+        outputFields.add(new TextField(computeClassLabel(inputTuple)));
+        
+        return new Tuple(outputSchema, outputFields);
+        
+	}
+
+	private String computeClassLabel(Tuple inputTuple) {
+        String inputText = inputTuple.<IField>getField(predicate.getInputAttributeName()).getValue().toString();
+        String className;
+        
+        try{
+            String path = Utils.getResourcePath("Python", TextdbProject.TEXTDB_EXP);
+            path += PYTHONSCRIPT;
+            String trainner = Utils.getResourcePath("Senti.pickle", TextdbProject.TEXTDB_EXP);
+            List<String> args = new ArrayList<String>(Arrays.asList(PYTHON, path, trainner));
+
+//            List<String> args = new ArrayList<String>(Arrays.asList(PYTHON, path));
+            args.addAll(new ArrayList<String>(Arrays.asList(inputText)));
+            
+            ProcessBuilder pb = new ProcessBuilder(args);
```

Comment:
Add a comment: "In order to use the NLTK package to do classification, we start a new process to run the package, and wait for the result of running the process as the class label of this text field."